### ISSUE TITLE
Double nickname change crashes BitBot

### ISSUE THREAD

**Issue by jesopo** [bitbot-irc/bitbot#51]
```
2019-05-12T19:05:26.706Z [CRITICAL] Unhandled exception: 'bitbottest'
Traceback (most recent call last):
  File "./start.py", line 97, in <module>
    bot.run()
  File "/home/cloud/src/python/bitbot/src/IRCBot.py", line 237, in run
    sock.parse_data(piece)
  File "/home/cloud/src/python/bitbot/src/IRCServer.py", line 212, in parse_data
    self.check_users()
  File "/home/cloud/src/python/bitbot/src/IRCServer.py", line 216, in check_users
    self.remove_user(user)
  File "/home/cloud/src/python/bitbot/src/IRCServer.py", line 181, in remove_user
    del self.users[user.nickname_lower]
KeyError: 'bitbottest'
```

**Comment by jesopo** [comment]
fixed by jesopo/bitbot@3e9d628d

**Comment by jesopo** [comment]
a more correct fix for this is in jesopo/bitbot@d0573706
